statsmodels/stats/oneway.py

```diff
@@ -846,8 +1049,8 @@ def simulate_power_equivalence_oneway(means, nobs, equiv_margin, vars_=None,
     return res
 
 
-def test_scale_oneway(data, method='bfm', center='median', transform='abs',
-                      trim_frac_mean=0.1, trim_frac_anova=0.):
+def test_scale_oneway(data, method="bf", center="median", transform="abs",
+                      trim_frac_mean=0.1, trim_frac_anova=0.0):
```

Comment:
@bashtage Do you know what's wrong with the signature?

```
WARNING: error while formatting signature for statsmodels.stats.oneway.test_scale_oneway: Handler <function mangle_signature at 0x7f2f8bd1bef0> for event 'autodoc-process-signature' threw an exception


Extension error:
Handler <function mangle_signature at 0x7f2f8bd1bef0> for event 'autodoc-process-signature' threw an exception
Makefile:41: recipe for target 'html' failed
make: *** [html] Error 2
```

I don't see why this travis job fails
https://travis-ci.org/github/statsmodels/statsmodels/jobs/710953394